sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -43,6 +43,10 @@ internal partial class TelemetryItem
             }
 
             SetResourceSdkVersionAndIkey(roleName, roleInstance, instrumentationKey);
+            if (AzMonList.GetTagValue(ref monitorTags.MappedTags, "sampleRate") is float sampleRate)
```

Comment:
make sure we have fall back. i.e we need to anticipate this tag not being present, and then don't send sample rate.

Reply:
We do have a fall back in searializer.